test/test_torch.py

```diff
@@ -6016,6 +6016,12 @@ def _select_broadcastable_dims(self, dims_full=None):
                 dims_small = [ds] + dims_small
         return (dims_small, dims_large, dims_full)
 
+    # collected tests of ops that used scalar_check in Declarations.cwrap for
+    # correctness
+    def test_scalar_check(self, device):
+        zero_d = torch.randn((), device=device)
+        self.assertRaises(RuntimeError, lambda: torch._multinomial_alias_setup(zero_d))
```

Comment:
So we still raise an error when we used to raise one. Just a different error message?

Reply:
no, it's the same, I'm just demonstrating that the change is correct by ensuring you can't pass a scalar in.